Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -603,6 +604,91 @@ public protocol AppBackend: Sendable {
     /// ``showAlert(_:window:responseHandler:)``.
     func dismissAlert(_ alert: Alert, window: Window?)
 
+    /// Creates a sheet object (without showing it yet). Sheets contain View Content.
+    /// They optionally execute provied code on dismiss and
+    /// prevent users from interacting with the parent window until dimissed.
+    func createSheet() -> Sheet
+
+    /// Updates the content and appearance of a sheet
+    func updateSheet(
+        _ sheet: Sheet,
+        content: Widget,
```

Comment:
For views with a single child we generally take the widget in the creation method instead of the update method. Each view graph node is guaranteed to keep the same widget for its whole lifecycle so you can give the sheet creation method a reference the the sheet content's underlying widget before updating the widget with the sheets size (so there shouldn't be any circular dependencies).